CMakeLists.txt

```diff
@@ -515,6 +515,8 @@ OCV_OPTION(ENABLE_PYLINT              "Add target with Pylint checks"
 OCV_OPTION(ENABLE_FLAKE8              "Add target with Python flake8 checker"                    (BUILD_DOCS OR BUILD_EXAMPLES) IF (NOT CMAKE_CROSSCOMPILING AND NOT APPLE_FRAMEWORK) )
 OCV_OPTION(OPENCV_DISABLE_FILESYSTEM_SUPPORT "Disable filesystem support" OFF)
 
+OCV_OPTION(OPENCV_DISABLE_THREAD_SUPPORT    "Build the library without multi-threaded code."            OFF)
+
```

Comment:
Why are there options are placed near python linter options group?
Why is there extra spacing between them?

Reply:
> Why are there options are placed near python linter options group?

No meaningful reason for this. Please let me know where to move them. Do you want me to move the FILESYSTEM one in a separate patch?

> Why is there extra spacing between them?

Again, no reason for this :). I will remove the extra space after moving.

Reply:
It makes sense to move them (both) upper, right after `OPENCV_ENABLE_MEMALIGN`

Reply:
Done. I'll move the FILESYSTEM one separately, though.

Reply:
With the addition of #20137 , I consider this done.